libres/lib/enkf/enkf_serialize.cpp

```diff
@@ -249,15 +240,11 @@ void enkf_matrix_deserialize(void *__node_data, int node_size,
         if (active_size == node_size) { /* All elements active */
             int row_index;
             for (row_index = 0; row_index < active_size; row_index++)
-                node_data[row_index] =
-                    matrix_iget(A, row_index + row_offset, column);
+                node_data[row_index] = A(row_index + row_offset, column);
         } else {
-            int row_index;
-            int node_index;
-            for (row_index = 0; row_index < active_size; row_index++) {
-                node_index = active_list[row_index];
-                node_data[node_index] =
-                    matrix_iget(A, row_index + row_offset, column);
+            for (int row_index = 0; row_index < active_size; row_index++) {
+                int node_index = active_list[row_index];
+                node_data[node_index] = A(row_index + row_offset, column);
```

Comment:
Seems like we could get rid of some code duplication here?

Reply:
forking outside for loop instead of inside..